frontend/src/components/TerminalGrid.tsx

```
@@ -0,0 +1,204 @@
+    );
+  };
+
+  return (
+    <div className="terminal-grid-wrapper">
+      <div className="terminal-grid-toolbar">
+        <GridSelector current={team.grid_layout} onChange={handleLayoutChange} />
+        <div className="terminal-grid-actions">
+          {addingAgent ? (
+            <div className="add-agent-input">
+              <input
+                autoFocus
```

Biome (2.3.14)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)